PokeDemo/Controllers/PokemonListViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+class PokemonListViewController: BaseViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var tableView: UITableView!
```

Comment:
**question:** 
1. why weak?
2. Can it be private?

Reply:
Covered in feda558a80950ca51ca6477354446339522b97cd
It is weak as outlets should lose memory when view is gone